dist/user-management-auth.js

```diff
@@ -121,17 +169,10 @@
       return ehaUserManagementAuthService.getCurrentUser()
         .then(function(user) {
           if (user && !user.isAdmin() && !user.hasRole(roles)) {
```

Comment:
Well here i just kept the former semantics when migrating Polio to the User Management. Yes, this logic says that an admin will result as always having all roles